src/main/resources/templates/tutorials.html

```diff
@@ -0,0 +1,146 @@
+
+                        <div class="col-sm-6">
+                            <img preload="true" minimal-lightbox="images/createexperiment2.png" src="images/createexperiment2.png" width="500" height="315"/></div>
+                        <div class="col-sm-6">
+                            <div class="heading col-sm-12 col-sm-offset-1">
+                                <p>&nbsp;</p>
+                                <p> 1) After logging in, click on "Experiment" on the navigation bar at the top. </p>
+                                <p> 2) This page provides an overview of your experiments. You can view the status, details, as well as start, stop, terminate or create an experiment</p>
+                                <p> 3) To create an experiment, click on "Create Experiment".</p>
+                                <p> 4) To view details of an experiment, cick on "View".</p>
+                            </div>
+                        </div>
+                        <p>&nbsp;</p>
+                        <p>&nbsp;</p>
+
+                        <div class="col-sm-6">
+                            <img preload="true" minimal-lightbox="images/createexperiment1.png" src="images/createexperiment1.png" width="500" height="315"/></div>
+                        <div class="col-sm-6">
+                            <div class="heading col-sm-12 col-sm-offset-1">
+                                <p>&nbsp;</p>
+                                <p> 1) Fill in team info. </p>
+                                <p> 2) Fill in name of experiment. </p>
+                                <p> 3) Give a description of the experiment.</p>
+                                <p> 4) Select your preferred scenario from the drop down menu.</p>
+                                <p> 5) Click on submit.</p>
+                            </div>
+                        </div>
+                        <p>&nbsp;</p>
+                        <p>&nbsp;</p>
+                        <p>&nbsp;</p>
+                        <div class="col-sm-6">
+                            <img preload="true" minimal-lightbox="images/createexperiment3.png" src="images/createexperiment3.png" width="500" height="315"/></div>
+                        <div class="col-sm-6">
+                            <div class="heading col-sm-12 col-sm-offset-1">
+                                <p>&nbsp;</p>
+                                <p> 1) Clicking on "View" brings up this page with details on your experiment and instructions on how to access experiment nodes. The fully qualifies name can be found here as well. </p>
+
+                            </div>
+                        </div>
+                    </div>
+<hr/>
+                <h2>2) How to Create an Account </h2>
+                <div class="heading col-sm-12 ">
+
+                    <div class="col-sm-6">
+                        <img preload="true" minimal-lightbox="images/register11.png" src="images/register11.png" width="500" height="315"/></div>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add an "alt" attribute to this image. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)